### PR TITLE
SAM-1816: Calculated questions shows checkmark or crossmark correctly

### PR DESCRIPTION
**Pull request by rodriguezdevera**
The behavior of Calculated Questions has been redefined to mark each answer according to the actual response introduced by the student.

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java [hunk 1]
```diff
@@ -2344,50 +2344,37 @@ public int compare(AnswerIfc a1, AnswerIfc a2) {
...
-          if (datas == null || datas.isEmpty())
-          {
+          if (datas == null || datas.isEmpty()) {
               fbean.setIsCorrect(false);
           } else {
               for (ItemGradingData data : datas) {
-
-                  if ((data.getPublishedAnswerId()!=null) && (data.getPublishedAnswerId().equals(answer.getId())))
-                  {
+                  if ((data.getPublishedAnswerId() != null) && (data.getPublishedAnswerId().equals(answer.getId()))) {
```
Comment (juanjmerono) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java]: This line is not really changed.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java [hunk 2]
```diff
@@ -2344,50 +2344,37 @@ public int compare(AnswerIfc a1, AnswerIfc a2) {
...
                       fbean.setItemGradingData(data);
                       fbean.setResponse(FormattedText.convertFormattedTextToPlaintext(data.getAnswerText()));
-                      fbean.setIsCorrect(false);
-                      if (answer.getText() == null)
-                      {
+                      if (answer.getText() == null) {
```
Comment (juanjmerono) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java]: This line is not really changed.

**Comment by rodriguezdevera** [comment]
You are right. 
Fixed it

**Comment by juanjmerono** [comment]
LGTM if no objections I'm planning to merge this tomorrow.

**Pull request merged**